### PR TITLE
Floating Window Mode, CSV exports

### PR DESCRIPTION
**Pull request by RichardTea**
- Floating Window Mode
- Improved High-DPI support
- New Multi-Universe View
- CSV export of the Source tables in Universe View
- 64bit Windows installer

This also includes a draft cmake (Windows only) that is incomplete and requires properly merging with the existing cmake work.

Closes Issue #316

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+      $<$<CONFIG:RELEASE>:/DEBUG>
+      $<$<CONFIG:RELEASE>:/OPT:REF>
+      $<$<CONFIG:RELEASE>:/OPT:ICF>
+    )
+  endif()
+  
+  # Docs only available if this is the main app
+  find_package(Doxygen)
+  if(Doxygen_FOUND)
+    add_subdirectory(docs)
```
Comment (docsteer) [inline CMakeLists.txt:43]: This doesn't work as although we have a docs directory, it isn't for Doxygen.

Comment (RichardTea) [reply CMakeLists.txt:43]: Good point, I forgot to remove this section from my cmake template

File: src/ui/universeview.cpp [hunk 2]
```diff
@@ -145,201 +133,96 @@ void UniverseView::listenerStarted(int universe)
...
-    sourceChanged(source);
-}
-
-void UniverseView::sourceOffline(sACNSource *source)
-{
-    if (!m_listener)
-        return;
-    sourceChanged(source);
-}
+//void UniverseView::sourceOnline(sACNSource *source)
```
Comment (docsteer) [inline src/ui/universeview.cpp:143]: Large block of commented out code - can we just remove it?

Comment (RichardTea) [reply src/ui/universeview.cpp]: I'd missed that entirely. Thanks!

**Pull request merged**